src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -978,25 +997,31 @@ def _bigbird_block_rand_mask(
             start = i - 2
             end = i
             if i == 1:
-                rand_attn[i - 1, :] = np.random.permutation(middle_seq[2:last])[:r]
+                rand_attn = rand_attn.at[i - 1].set(jax.random.permutation(indices_prng_key, middle_seq[2:last])[:r])
```

Comment:
Nice! We could split this over two lines for readability:
```suggestion
                seq_values = jax.random.permutation(indices_prng_key, middle_seq[2:last])[:r]
                rand_attn = rand_attn.at[i - 1].set(seq_values)
```

Reply:
Yeah, we definitely should :)